great_tables/_scss.py

```diff
@@ -39,23 +39,30 @@
 )
 
 
-def font_color(color: str, table_font_color: str, table_font_color_light: str):
-    if color.startswith("#"):
-        rgb = wc.hex_to_rgb(color)
-    elif color.startswith("rgb") and "%" in color:
-        # TODO: rgb_percent_to_rgb() expects a tuple
-        raise NotImplementedError()
-        rgb = wc.rgb_percent_to_rgb(color)
-    else:
-        rgb = wc.name_to_rgb(color)
+def _font_color(color: str, table_font_color: str, table_font_color_light: str) -> str:
```

Comment:
This `_font_color()` function was reworked for two reasons:

1.  we already had some existing logic for determining which opposing color (choice between a predetermined `dark` and `light` color) should be used for maximal contrast (that was used in `data_color()`)
2. we documented that all CSS3 and even X11 color names would be accepted as named colors. We didn't include other CSS color keywords like `"transparent"` and `"currentColor"`. The webcolors package doesn't support those and we already have `_html_color()` which accepts them handles the normalization to hex color as well.